### PR TITLE
style(3000): Use new Central icons

### PR DESCRIPTION
**Pull request by Twixes**
## Changes

Starting to wean off Material icons and using [Central](https://iconists.co/central). This updates the 3000 navigation to use new icons from `@posthog/icons`.
| Before | After |
| --- | --- |
| before | after |

(the annotation and events ones aren't the final ones yet, need to add those to `@posthog/icons`)

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.0'
+lockfileVersion: '6.1'
```
Comment (daibhin) [inline pnpm-lock.yaml:1]: Never sure what this flopping means 🤷

**Review by daibhin — COMMENTED** [review]
Two snapshots seem to look like the icons are huge

`frontend/__snapshots__/posthog-3000-navigation--dark-mode.png`
`frontend/__snapshots__/posthog-3000-navigation--light-mode.png`

File: pnpm-lock.yaml (hunk 1 above)
Comment (Twixes) [reply pnpm-lock.yaml:1]: I think _some_ people (*`git blame`s* – in this case, @pauldambra – *judgmental look*) have a slightly older version of `pnpm` installed, so we go back to 6.0. But they're the same, so this doesn't matter at all

**Comment by Twixes** [comment]
Oops, fixed! 😄 @daibhin

**Review by daibhin — APPROVED** [review]

**Pull request merged**